docs/components/modeler/bpmn/bpmn.md

```diff
@@ -0,0 +1,49 @@
+---
+id: modeler-bpmn
+title: BPMN in Modeler
+description: Let's start modeling by creating a new BPMN diagram.
+---
+
+# Create new BPMN 2.0 diagram
+
+To start modeling, create a new BPMN 2.0 diagram by selecting **Create New Diagram** in the **Diagrams** tab for your process engine in the top-level menu.
+
+If working in Platform, select **Create diagram > Create new BPMN diagram**.
```

Comment:
Remove this line.

Reply:
Errrr.... maybe move this into an admonition and fully qualify Platform -> Camunda Platform.